olive/passes/pytorch/lora.py

```diff
@@ -114,43 +116,34 @@ def create_training_args(self) -> transformers.TrainingArguments:
         return transformers.TrainingArguments(**args, **extra_args)
 
 
-class QLoRA(Pass):
-    """Run QLoRA fine-tuning on a Hugging Face PyTorch model.
-
-    See https://arxiv.org/abs/2305.14314 for more details on the method.
-
-    This pass only supports PyTorchModel with hf_config.
-    """
+class LoRABase(Pass):
+    """Base class for LoRA and QLoRA fine-tuning passes."""
 
     @staticmethod
     def _default_config(accelerator_spec: AcceleratorSpec) -> Dict[str, PassConfigParam]:
         return {
-            # quantization parameters
-            "compute_dtype": PassConfigParam(
```

Comment:
I think this might still be useful for lora training. It can be used to set `torch_dtype` in the `model_loading_args` so that we can train the model in fp16 or bf16 if needed. 
Could rename the parameter to `torch_dtype` or something and update the description: data type for model weights and adapter weights. For 4bit quantized model, it is also the computation data type for the quantized modules.

Reply:
Sounds good to me. Will update it.